static/app/views/alerts/rules/metric/types.tsx

```diff
@@ -154,10 +154,7 @@ export enum TimePeriod {
   SIX_HOURS = '6h',
   ONE_DAY = '1d',
   THREE_DAYS = '3d',
-  // Seven days is actually 10080m but Snuba can only return up to 10000 entries, for this
-  // we approximate to 9998m which prevents rounding errors due to the minutes granularity
-  // limitations.
-  SEVEN_DAYS = '9998m',
+  SEVEN_DAYS = '7d',
```

Comment:
This is what's breaking the tests currently. Can try to add more hacks to workaround the limitation, but probably worth asking if this is still needed?

Reply:
+1 about asking, unfortunately I don't have context over it.

Reply:
Decided to revert this change just to be safe.